corehq/apps/commtrack/sms.py

```diff
@@ -194,10 +194,10 @@ def get_supply_point_and_location(self, loc_code):
     def product_from_code(self, prod_code):
         """return the product doc referenced by prod_code"""
         prod_code = prod_code.lower()
-        p = Product.get_by_code(self.domain.name, prod_code)
-        if p is None:
+        try:
+            return SQLProduct.objects.get(domain=self.domain.name, code__iexact=prod_code)
```

Comment:
is it necessary to use `iexact`? `prod_code` is converted to lower case 2 lines up.